.flake8

```diff
@@ -0,0 +1,9 @@
+[flake8]
+# https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/
+ignore = E203, E266, E501, W503, F403, F401 , E402
+max-line-length = 79
+max-complexity = 15
+select = B,C,E,F,W,T4,B9
```

Comment:
What was the reasoning behind the `max-line-length, max-complexity` and `select` configurations? Are these agreed upon standards at KBase or common industry standards?

Reply:
@eamahanna I just used whatever was in that blog post linked and what I had set up for ee2. Some of these are required for BLACK to work, as black ignores some PEP8 syntax.

Reply:
Removed complexity from checks